docs/en/integrations/what-is-an-integration.asciidoc

```diff
@@ -31,9 +31,9 @@ All packages must adhere to the <<package-spec,package specification>> -- a form
 
 . Publish the integration to the package registry
 +
-Once an integration (package) has been created, it needs to be built. Built integrations are stored in the {package-storage-repo-link}[Package Storage repository] and served up via the {package-registry-repo-link}[Elastic Package Registry (EPR)].
+Once an integration (package) has been created, it needs to be built. Built integrations are stored in the {package-storage-repo-link}[Package Storage repository] and served up via the {package-registry-repo-link}[{package-registry}].
 The {fleet} UI in {kib} connects to the {package-registry} and allows users to discover, install, and configure Elastic Packages.
-The EPR can also be run locally for testing purposes.
+The {package-registry} can also be run locally for testing purposes, or <<air-gapped-diy-epr,deployed on-premise in air-gapped environments>>.
```

Comment:
Can we remove "for testing purposes", now that we are officially supporting it as a GA product?

Reply:
I left this because this section made reference to package development, but yes, better to remove this sentence to avoid confusion.